### PR TITLE
#243 Print additionnal point in curves output file when there is a ch…

### PR DESCRIPTION
**Pull request by rosiereflo**
…ange in discrete variables

### CODE REVIEW

File: dynawo/sources/API/CRV/CRVCurve.h [hunk 1]
```diff
@@ -36,6 +36,16 @@ class Point;
...
  public:
+/**
+* defined type of variables
+*/
+typedef enum {
+  UNDEFINED,
+  CALCULATED_VARIABLE,
+  DISCRETE_VARIABLE,
+  CONTINUOUS_VARIABLE
+} CurveType_t;  ///< type on constraint
```
Comment (adriguir) [inline dynawo/sources/API/CRV/CRVCurve.h]: Adapt comment

File: dynawo/sources/API/CRV/CRVCurveImpl.h [hunk 2]
```diff
@@ -183,7 +193,8 @@ class Curve::Impl : public Curve {
   double *buffer_;  ///< adress buffer where to find value
   std::vector<boost::shared_ptr<Point> > points_;  ///< vector of each values
   bool isParameterCurve_;  ///< @b true if a parameter curve, @b false if variable
-  bool isCalculatedVariableCurve_;  ///< @b true if a calculated variable curve, @b false if variable
+  CurveType_t curveType_;  ///< @b true if a calculated variable curve, @b false if variable
```
Comment (adriguir) [inline dynawo/sources/API/CRV/CRVCurveImpl.h]: Adapt comment too

File: dynawo/sources/API/CRV/CRVCurve.h (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/API/CRV/CRVCurve.h]: solved

File: dynawo/sources/API/CRV/CRVCurveImpl.h (hunk 2 above)
Comment (rosiereflo) [reply dynawo/sources/API/CRV/CRVCurveImpl.h]: solved

**Pull request merged**